mcos/mcos.py

```diff
@@ -28,3 +29,11 @@ def simulate_optimizations(obs_simulator: ObservationSimulator, n_sims: int, opt
             'stdev': np.std(error_estimator[optimizers.name])
         } for optimizer in optimizers
     ]).set_index('optimizer')
+
+
+def optimal_allocation(mu: np.array, cov: np.array):
```

Comment:
We should remove this. This is just one type of portfolio optimization. It does the same thing as `optimizer.allocate`

Reply:
The "optimal" allocation should be obtained by taking running the optimizer using the original covariance matrix

Reply:
Aren't I already doing this here `optimizer.allocate(mu_hat, cov_hat)`? I'm not understanding the difference between calculating the allocation and "optimal" allocation.